stan/math/opencl/opencl_context.hpp

```diff
@@ -32,35 +32,7 @@
  */
 namespace stan {
 namespace math {
-namespace opencl {
-/**
- * A helper function to convert an array to a cl::size_t<N>.
- * This implementation throws because cl::size_t<N> for N!=3
- * should throw.
- *
- * @param values the input array to be converted
- * @return the cl::size_t<N> converted from the input array
- */
-template <int N>
-inline cl::size_t<N> to_size_t(const size_t (&values)[N]) {
-  throw std::domain_error("cl::size_t<N> is not supported for N != 3");
-}
 
-/**
- * A template specialization of the helper function
- * to convert an array to a cl::size_t<3>.
- *
- * @param values the input array to be converted
- * @return the cl::size_t<3> converted from the input array
- */
-template <>
-inline cl::size_t<3> to_size_t(const size_t (&values)[3]) {
-  cl::size_t<3> s;
-  for (size_t i = 0; i < 3; i++)
-    s[i] = values[i];
-  return s;
-}
-}  // namespace opencl
```

Comment:
:raised_hands: :raised_hands: :raised_hands:

I hated that we had to do this before. Finally its gone.

Reply:
lol same. Part of updating these was because I wanted to zip that out